LSP-html.sublime-settings

```diff
@@ -13,18 +12,120 @@
 		},
 	],
 	"initializationOptions": {
-		"provideFormatter": true,
 		"embeddedLanguages": {
 			"css": true,
-			"html": true,
 			"javascript": true,
 		},
+		"dataPaths": [],
+		"provideFormatter": false,
```

Comment:
So should it be `true` then?

Reply:
yeah. i think formatting is a common and useful functionality.